src/App.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import Template from '../src/components/template'
+import Parser from '../src/classes/parser/index'
```

Comment:
não precisa colocar o index
```suggestion
import Parser from '../src/classes/parser'
```